src/javascripts/legacy_app.js

```diff
@@ -41,6 +42,12 @@ var App = {
 
   logClosedModal: function() {
     console.log('Closed a modal.');
+  },
+
+  storeClosedAt: function() {
+    var closedAt = new Date();
+    console.log('CLOSING', closedAt);
+    this.store('lastClosedAt', closedAt);
```

Comment:
I'm not sure what the issue is here. This PR adds an event that belongs only in the master branch for backward compatibility. The PR was updated so that the handler for that event just calls `console.log`. We aren't going to add `$(window).on('unload', function() {})` to from-scratch or demo_apps are we?